api/v2/api.go

```diff
@@ -424,7 +424,7 @@ func (api *API) getAlertGroupsHandler(params alertgroup_ops.GetAlertGroupsParams
 	return alertgroup_ops.NewGetAlertGroupsOK().WithPayload(res)
 }
 
-func (api *API) alertFilter(matchers []*labels.Matcher, silenced, inhibited, active bool) func(a *types.Alert, now time.Time) bool {
+func (api *API) alertFilter(matchers []*labels.Matcher, silenced, inhibited, muted, active bool) func(a *types.Alert, now time.Time) bool {
```

Comment:
nit: You've got it right on the places that this used but I feel that is a lot less risky if you add the new parameter at the end as opposed to before the last one.

Reply:
I can change it 🙂 It sounds like we don't have sufficient test coverage to have confidence in this?